packages/gasket-plugin-swagger/lib/index.js

```diff
@@ -123,13 +123,19 @@ const plugin = {
         });
       }
 
+      context.gasketConfig
+        .addImport('{ createRequire }', 'module')
+        .addExpression('const require = createRequire(import.meta.url);')
+        .addExpression("const { version } = require('./package.json')")
+        .injectValue('swagger.jsdoc.definition.info.version', 'version');
```

Comment:
Instead of surfacing all this in gasket config, it would be preferable to just handle in the plugin code during build, perhaps [here](https://github.com/godaddy/gasket/blob/39be7c1fda800ecc7834fc07bf54e51b4afcd254/packages/gasket-plugin-swagger/lib/build-swagger-definition.js#L14).